ua/extension_object.go

```diff
@@ -127,6 +127,16 @@ func ExtensionObjectTypeID(v interface{}) *ExpandedNodeID {
 	case *ServerStatusDataType:
 		return NewFourByteExpandedNodeID(0, id.ServerStatusDataType_Encoding_DefaultBinary)
 	default:
-		return NewTwoByteExpandedNodeID(0)
+		t := eotypes.Lookup(v)
```

Comment:
We should refactor the `TypeRegistry` to use `NodeID`s as keys so that we don't have to ignore the error below. I'll add a patch for this.